cmd/ddev/cmd/commands_test.go

```diff
@@ -194,7 +194,7 @@ func TestCustomCommands(t *testing.T) {
 	assert.NoError(err)
 
 	// Make sure that all the official ddev-provided custom commands are usable by checking help
-	for _, c := range []string{"launch", "xdebug"} {
+	for _, c := range []string{"launch", "mailpit", "xdebug"} {
```

Comment:
This fails, but the failure isn't related to this PR so I'll remove the addition.